tmail-backend/webadmin/webadmin-rate-limit/src/main/scala/com/linagora/tmail/webadmin/RateLimitPlanManagementRoutes.scala

```diff
@@ -103,12 +103,12 @@ class RateLimitPlanManagementRoutes @Inject()(planRepository: RateLimitationPlan
 
   private def toCreateRequest(request: Request): RateLimitingPlanCreateRequest = {
     val createRequestDTO = createRequestExtractor.parse(request.body())
-    val combinedLimitations = util.List.of(createRequestDTO.getTransitLimits.map(toTransitLimitation).toScala,
-      createRequestDTO.getDeliveryLimits.map(toDeliveryLimitation).toScala,
-      createRequestDTO.getRelayLimits.map(toRelayLimitation).toScala)
-      .stream()
-      .filter(_.isDefined)
-      .map(_.get)
+    val combinedLimitations = java.util.stream.Stream.of(
```

Comment:
Maybe `OperationLimitations.liftOrThrow()` is your want?
then, we won't care about "what is operationLimitationName", and we can merging `toTransitLimitation/toRelayLimitation/toDeliveryLimitation` to a single method. 

// if yes, we need to refactor this API to accept `equalIgnoreCase`